src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java

```diff
@@ -67,6 +69,22 @@ public RightClickMenu(JabRefFrame frame, BasePanel panel) {
         copySpecialMenu.add(new GeneralAction(Actions.COPY_CITE_KEY, Localization.lang("Copy \\cite{BibTeX key}"), KeyBinding.COPY_CITE_BIBTEX_KEY));
         copySpecialMenu.add(new GeneralAction(Actions.COPY_KEY_AND_TITLE, Localization.lang("Copy BibTeX key and title"), KeyBinding.COPY_BIBTEX_KEY_AND_TITLE));
         copySpecialMenu.add(new GeneralAction(Actions.COPY_KEY_AND_LINK, Localization.lang("Copy BibTeX key and link"), KeyBinding.COPY_BIBTEX_KEY_AND_LINK));
+
+        // the submenu will behave dependent on what style is currently selected (citation/preview)
+        PreviewPreferences previewPreferences = Globals.prefs.getPreviewPreferences();
+        String style = previewPreferences.getPreviewCycle().get(previewPreferences.getPreviewCyclePosition());
+        if (CitationStyle.isCitationStyleFile(style)) {
+            JMenu copyCitationMenu = new JMenu(Localization.menuTitle("Copy citation..."));
+            copyCitationMenu.add(new GeneralAction(Actions.COPY_CITATION_HTML, "HTML", KeyBinding.COPY_PREVIEW));
```

Comment:
Can we move this action one level up so that independent of the current preview mode, always "Copy preview" is shown and that this always copies HTML?

Reason: The shortcut of this action is also the same - independent of the mode. Thus, it should also be in the same place.

The other entries can be left as is.